oap-server/server-starter/src/main/resources/application.yml

```diff
@@ -36,6 +36,10 @@ cluster:
 #  nacos:
 #    serviceName: ${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
 #    hostPort: ${SW_CLUSTER_NACOS_HOST_PORT:localhost:8848}
+#  etcd:
```

Comment:
There are two application.yml needed to be changed. I think you miss one#

Reply:
I've added.